pages/[address].tsx

```diff
@@ -78,9 +78,16 @@ export const getStaticProps: GetStaticProps = async (ctx) => {
 
   const queryClient = new QueryClient();
   const cmsSynths = await contentfulClient.getAllSynths();
+  const client = constructClient(cmsSynth.chainId);
 
   await queryClient.prefetchQuery("all synths", async () =>
-    (await Promise.all(cmsSynths.map(fetchCompleteSynth))).filter(errorFilter)
+    (
+      await Promise.all(
+        cmsSynths.map((synth) =>
+          constructClient(synth.chainId).fetchCompleteSynth(synth)
+        )
+      )
+    ).filter(errorFilter)
```

Comment:
im wondering if we need to change filtering errors if we have custom synths in contentful? things that wont have representation in the api, like across? maybe requires a separate pr. wdyt?

Reply:
This can be added in a separate PR. Filtering errors is a good ideea because in some cases errors aren't critical and we can display the Contentful item partially, instead of hiding it completely.

Reply:
Mhh, I'm actually not sure how something like Across would be handled after the error. But I guess we can avoid worrying about it for now as the whole page would likely need to change (as we only support EMPs and LSPs for now) if we want to support it